### DISCUSSION TITLE
How to Check if BR_2 forwards the multicast ping packet to the LAN?

### DISCUSSION THREAD

**Discussion by xiaojiang2017** [openthread/openthread#9118] [Q&A] [answered]
**Describe the bug** A clear and concise description of what the bug is.
Hi, We are testing `Test 5.10.7 MATN TC 09`. But always failed to verify that the DUT sent a properly formatted Ping Request. I compared the two sniffer data and did not see any difference. Could you help me with how to do the analysis? Thanks.
![image](https://github.com/openthread/openthread/assets/73470725/83078e51-5ae6-473d-8d4e-142c2ca11c57)
![image](https://github.com/openthread/openthread/assets/73470725/c218c4b8-2ec7-449e-a099-8e0ae29a07bc)

[Test 5.10.7.zip](https://github.com/openthread/openthread/files/11621587/Test.5.10.7.zip)

1. Git commit id GSDK v4.2.3
2. IEEE 802.15.4 hardware platform EFR32MG21

**Comment by jwhui** [comment]
At first glance, the ICMPv6 packet seems OK.

Did you solicit feedback from Thread Test Harness team on why the packet you captured was not accepted?

**Comment by sunytt** [comment, marked as answer]
It looks similar to issue 1 in:  https://threadgroup.atlassian.net/browse/DEV-2381

**Reply by xiaojiang2017** [reply]
Hi,
I think this is a bug of Thread Harness in manual testing. Thread Harness will prompt multiple MAC addresses every time. After the user clicks, the correct DUT MAC address is not recognized.
![8f896d45303b06d3fc6c4d332d20831](https://github.com/openthread/openthread/assets/73470725/7ca17441-4973-4a6e-8286-e073fd15d3f8)
![4c16f1b7b45d0535bbd608e4c193877](https://github.com/openthread/openthread/assets/73470725/33779c9f-8f88-46e0-8064-cbde582fc176)
If I directly set the DUT MAC address, it will not prompt again, and finally, the verification passes. Any thoughts?
![cc73f9d6d0e1bcb3ec0d41da136078f](https://github.com/openthread/openthread/assets/73470725/3e00d7a9-4575-48e8-95df-de6455de4a76)

**Reply by jwhui** [reply]
That seems to be consistent with the conclusion in [DEV-2381](https://threadgroup.atlassian.net/browse/DEV-2381).

**Reply by xiaojiang2017** [reply]
Hi @jwhui ,
I have no access to [DEV-2381](https://threadgroup.atlassian.net/browse/DEV-2381). Could you provide more information?

**Reply by jwhui** [reply]
The link is limited to Thread Group members. Send email to help@threadgroup.org to request access.
